doc/manuals/work_queue/index.md

```diff
@@ -1378,6 +1378,54 @@ warranted:
     }
     ```
 
+### Fetching Input Data via URL
+
+Tasks can fetch remote data named by a URL into the worker's cache.
+For example, if you have a large dataset provided by a web server,
+use `specify_url` to attach the URL to a local file.  The data
+will be downloaded once per worker and then shared among all
+tasks that require it:
+
+
+=== "Python"
+    ```python
+    t.specify_url("http://somewhere.com/data.tar.gz", "data.tar.gz", cache=True)
+    ```
+
+=== "Perl"
+    ```python
+    $t->specify_url("http://somewhere.com/data.tar.gz", "data.tar.gz", flags=wq.WORK_QUEUE_CACHE)
+    ```
+
+=== "C"
+    ```python
+    work_queue_task_specify_url(t,"http://somewhere.com/data.tar.gz", "data.tar.gz", WORK_QUEUE_CACHE)
```

Comment:
Does it also work for outputs? Does it need an explicit WORK_QUEUE_INPUT?

Reply:
I think we shouldn't promise more than we know how to deliver at this point.  Handling output files means sending along credentials of some kind, and that means a different API and RPC.  What do you think if I rename these to specify_input_url()and specify_input_file_command() to be more clear?

Reply:
I like that idea.

Reply:
Ah, I had forgotten that our old deprecated API was specify_{input/output}_{file/buffer/piece}_{no cache} which we consolidated into specify_{file/buffer/piece} with flags.  To be consistent here, we should stick with specify_{url/command} and throw an error on output types.

Reply:
Clarified documentation and examples.